tests/parametric/test_telemetry.py

```diff
@@ -125,53 +124,25 @@ def test_library_settings(self, library_env, test_agent, test_library):
-            ("trace_http_server_error_statuses", "500"),
-            ("trace_http_client_tag_query_string", ("true", True)),
-            (
-                "trace_client_ip_header",
-                "X-Forwarded-For",
-            ),  # Unclear if correct key, see: https://docs.google.com/document/d/1kI-gTAKghfcwI7YzKhqRv2ExUstcHqADIWA4-TZ387o/edit?disco=AAABVcOUNfU
-        ]:
-            if context.library == "cpp" and apm_telemetry_name in ("trace_header_tags"):
-                continue
-            apm_telemetry_name = _mapped_telemetry_name(context, apm_telemetry_name)
-            cfg_item = configuration_by_name.get(apm_telemetry_name)
-            assert cfg_item is not None, "Missing telemetry config item for '{}'".format(apm_telemetry_name)
-            if isinstance(value, tuple):
-                assert cfg_item.get("value") in value, "Unexpected value for '{}'".format(apm_telemetry_name)
-            else:
-                assert cfg_item.get("value") == value, "Unexpected value for '{}'".format(apm_telemetry_name)
-            assert cfg_item.get("origin") == "env_var", "Unexpected origin for '{}'".format(apm_telemetry_name)
 
-        # Golang and CPP do not support DD_TRACE_<INTEGRATION>_ENABLED, so don't test them for this config.
-        apm_telemetry_name = _mapped_telemetry_name(context, "trace_disabled_integrations")
-        cfg_item = configuration_by_name.get(apm_telemetry_name)
-        if (
-            context.library == "java"
-            or context.library == "dotnet"
-            or context.library == "node"
-            or context.library == "python"
-            or context.library == "ruby"
-        ):
-            assert cfg_item is not None, "Missing telemetry config item for '{}'".format(apm_telemetry_name)
-            assert cfg_item.get("value") is "grpc"
-        if context.library == "php":
-            assert cfg_item is not None, "Missing telemetry config item for '{}'".format(apm_telemetry_name)
-            assert cfg_item.get("value") is "phpredis"
-        # The trace_agent_url is a container address -- don't know the value, but we can assert its not empty (i.e, that it reports)
-        apm_telemetry_name = _mapped_telemetry_name(context, "trace_agent_url")
-        cfg_item = configuration_by_name.get(apm_telemetry_name)
-        assert cfg_item is not None, "Missing telemetry config item for '{}'".format(apm_telemetry_name)
+        # Check that the tags name match the expected value
+        assert configuration_by_name.get("DD_ENV").get("value") == "dev"
+        assert configuration_by_name.get("DD_SERVICE").get("value") == "service_test"
+        assert configuration_by_name.get("DD_VERSION").get("value") == "5.2.0"
+        assert configuration_by_name.get("DD_TRACE_RATE_LIMIT").get("value") == 10
+        assert (
+            configuration_by_name.get("DD_TRACE_HEADER_TAGS").get("value") == "User-Agent:my-user-agent,Content-Type."
```

Comment:
Is case sensitivity an issue here? E,g, `User-Agent` vs `user-agent`?

Reply:
Nah it shouldn't be based on the RFC, I just tried pulling the same values for the configs that is recommended for the others tests in the RFC(swap some around to change them from the defaults):
https://docs.google.com/document/d/1kI-gTAKghfcwI7YzKhqRv2ExUstcHqADIWA4-TZ387o/edit#heading=h.e0s28phvgd5g